dcpy/connectors/github.py

```diff
@@ -1,25 +1,39 @@
+from __future__ import annotations
```

Comment:
This is so that a `classfunction` can return (typed) its own class. I'll comment below on if we even want that